src/controllers/planter.controller.ts

```diff
@@ -74,14 +74,15 @@ export class PlanterController {
     return await this.planterRepository.findById(id);
   }
 
-  @get('/planter/{id}/selfies', {
+  @get('/organization/{organizationId}/planter/{id}/selfies', {
```

Comment:
This endpoint is already implemented in `planterOrganization.controller.ts`.

Reply:
The existing endpoint wasn't responding to the "/selfies" request.  I can add the /selfies endpoint with the org id to the planterOrganization.controller.ts and keep the non-org endpoint on planter.controller.